### DISCUSSION TITLE
this version 	1.1.223 not is posible install in curren version visual studio

### DISCUSSION THREAD

**Discussion by juanmarquezruiz** [FortuneN/FineCodeCoverage#453] [General]
4/09/2024 3:59:54 p. m. - Microsoft VSIX Installer
4/09/2024 3:59:54 p. m. - -------------------------------------------
4/09/2024 3:59:54 p. m. - vsixinstaller.exe version:
4/09/2024 3:59:54 p. m. - 17.10.2186+0bf64ad641
4/09/2024 3:59:54 p. m. - -------------------------------------------
4/09/2024 3:59:54 p. m. - Command line parameters:
4/09/2024 3:59:54 p. m. - C:\Program Files (x86)\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\VSIXInstaller.exe,C:\Users\jmarquez35\OneDrive - DXC Production\Documents\FineCodeCoverage2022.vsix
4/09/2024 3:59:54 p. m. - -------------------------------------------
4/09/2024 3:59:54 p. m. - Microsoft VSIX Installer
4/09/2024 3:59:54 p. m. - -------------------------------------------
4/09/2024 3:59:55 p. m. - Initializing Install...
4/09/2024 3:59:55 p. m. - Buscando productos aplicables...
4/09/2024 3:59:56 p. m. - Producto instalado encontrado: Visual Studio Enterprise 2022
4/09/2024 3:59:56 p. m. - Producto instalado encontrado: Ubicación global
4/09/2024 3:59:56 p. m. - Detalles de la extensión...
4/09/2024 3:59:56 p. m. - 	Identifier         : FineCodeCoverage2022.ad53fb28-7a11-4465-a27a-3550499ea4a1
4/09/2024 3:59:56 p. m. - 	Name               : Fine Code Coverage
4/09/2024 3:59:56 p. m. - 	Author             : Fortune Ngwenya
4/09/2024 3:59:56 p. m. - 	Version            : 1.1.223
4/09/2024 3:59:56 p. m. - 	Description        : Visualize unit test code coverage easily for free in Visual Studio Community Edition (and other editions too)
4/09/2024 3:59:56 p. m. - 	Locale             : en-US
4/09/2024 3:59:56 p. m. - 	MoreInfoURL        : https://marketplace.visualstudio.com/items?itemName=FortuneNgwenya.FineCodeCoverage
4/09/2024 3:59:56 p. m. - 	InstalledByMSI     : False
4/09/2024 3:59:56 p. m. - 	SupportedFrameworkVersionRange : [4.7,)
4/09/2024 3:59:56 p. m. - 	SignatureState     : Unsigned
4/09/2024 3:59:56 p. m. - 	Productos admitidos : 
4/09/2024 3:59:56 p. m. - 		Microsoft.VisualStudio.Community
4/09/2024 3:59:56 p. m. - 			Version : [17.10,18.0)
4/09/2024 3:59:56 p. m. - 			ProductArchitecture : amd64
4/09/2024 3:59:56 p. m. - 		Microsoft.VisualStudio.Community
4/09/2024 3:59:56 p. m. - 			Version : [17.10,18.0)
4/09/2024 3:59:56 p. m. - 			ProductArchitecture : arm64
4/09/2024 3:59:56 p. m. - 	Referencias        : 
4/09/2024 3:59:56 p. m. - 	Requisitos previos : 
4/09/2024 3:59:56 p. m. - 		-------------------------------------------------------
4/09/2024 3:59:56 p. m. - 		Identifier   : Microsoft.VisualStudio.Component.CoreEditor
4/09/2024 3:59:56 p. m. - 		Name         : Visual Studio core editor
4/09/2024 3:59:56 p. m. - 		Version      : [17.10,18.0)
4/09/2024 3:59:56 p. m. - Detalles de la firma...
4/09/2024 3:59:56 p. m. - 	Extension is not signed.
4/09/2024 3:59:56 p. m. - VSIXInstaller.NoApplicableSKUsException: Esta extensión no se puede instalar en ninguno de los productos instalados actualmente.
   en VSIXInstaller.ExtensionService.GetInstallableDataImpl(String vsixPath, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   en VSIXInstaller.ExtensionService.GetInstallableData(String vsixPath, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   en VSIXInstaller.ExtensionPackService.IsExtensionPack(IStateData stateData, Boolean isRepairSupported)
   en VSIXInstaller.ExtensionPackService.ExpandExtensionPackToInstall(IStateData stateData, Boolean isRepairSupported)
   en VSIXInstaller.App.Initialize(Boolean isRepairSupported)
   en VSIXInstaller.App.Initialize()
   en System.Threading.Tasks.Task`1.InnerInvoke()
   en System.Threading.Tasks.Task.Execute()

**Comment by tonyhallett** [comment]
You need to install in visual studio at the necessary version. Update visual studio.

Did you get the vsix from GitHub or the marketplace?
